code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -479,7 +479,10 @@
 		return
 
 	if(!can_be_inserted(W))
-		return TRUE
+		if(istype(W, /obj/item/weapon/glue))
```

Comment:
This checks seems like it should be the opposite. Glue has the more intuitive interaction (target container if full) whereas bolas have the unintuive interaction.

That's just my view though so I'm open to hearing other opinions.

Reply:
I really don't know of a good way to look for other cases.
The only other case I know about is fire extinguishers. Leaving it as is will make it so that attempting to put a fire extinguisher in a full bit of storage will not fire the extinguisher. Reversing the check will make it so that fire extinguishers will fire if the storage item is full. (of course, if the extinguisher can be put in the item, it will not fire regardless).

I don't think there are very many items that you want to apply to a storage item, like you can't splash beakers on them regardless of the orientation of the check. For that reason I would rather keep it as is.